core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala

```diff
@@ -861,6 +861,9 @@ class WorkspaceServiceSpec extends AnyFlatSpec with ScalatestRouteTest with Matc
     )
 
     val workspace = Await.result(services.mcWorkspaceService.createMultiCloudWorkspace(workspaceRequest), Duration.Inf)
+    assertResult(Option(workspace.toWorkspaceName)) {
```

Comment:
I thought `Option` meant that it could be None. Don't we want to check that it isn't None?

Reply:
`Option` can be `None`, but in this case since `workspace.toWorkspaceName` isn't null, `Option(workspace.toWorkspaceName)` is equivalent to `Some(workspace.toWorkspaceName)

Reply:
`Some` doesn't ensure that `workspace.toWorkspaceName` isn't null either. If `workspace.toWorkspaceName` is null, then `Some(workspace.toWorkspaceName)` will return `Some(null)` while `Option(workspace.toWorkspaceName)` will return `None` which I think is better